ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/review/v1/review/_cq_template.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Review"
+    fieldType="plain-text"/>
```

Comment:
fieldType should be defaulted and it should not be mandatory for review. We should get rid of fieldType from JCR and add it in the interface directly as fallback like we have done for other components

Reply:
Removed